extensions/interactions/NumberWithUnits/directives/NumberWithUnits.js

```diff
@@ -130,8 +130,16 @@ oppia.directive('oppiaResponseNumberWithUnits', [
         'number_with_units_response_directive.html'),
       controller: ['$scope', '$attrs', function($scope, $attrs) {
         var answer = HtmlEscaperService.escapedJsonToObj($attrs.answer);
-        $scope.answer = NumberWithUnitsObjectFactory.fromDict(
-          answer).toString();
+        // This check is added because for this specific interaction, it was
+        // found that at some points when this was loaded for the first time,
+        // the answer object was not in the right format. So, until it becomes
```

Comment:
Please elaborate: what do you mean by "not in the right format"? Is there an issue with repro instructions for how to reproduce this? It might point to a deeper bug.

Reply:
Added comment and filed issue.